src/Build/Resources/xlf/Strings.de.xlf

```diff
@@ -1979,47 +1979,47 @@ Auslastung:          {0} Durchschnittliche Auslastung: {1:###.0}</target>
       </trans-unit>
       <trans-unit id="EvaluationStarted">
         <source>Evaluation started ("{0}")</source>
-        <target state="new">Evaluation started ("{0}")</target>
-        <note></note>
+        <target state="translated">Evaluierung gestartet ("{0}")</target>
+        <note />
       </trans-unit>
       <trans-unit id="EvaluationFinished">
         <source>Evaluation finished ("{0}")</source>
-        <target state="new">Evaluation finished ("{0}")</target>
-        <note></note>
+        <target state="translated">Evaluierung abgeschlossen ("{0}")</target>
+        <note />
       </trans-unit>
       <trans-unit id="InvalidBinaryLoggerParameters">
         <source>MSB4234: The binary logger expects a single parameter (the output file name). {0} parameters were passed: "{1}".</source>
-        <target state="new">MSB4234: The binary logger expects a single parameter (the output file name). {0} parameters were passed: "{1}".</target>
+        <target state="translated">MSB4234: Die binäre Protokollierung erwartet einen einzelnen Parameter (den Ausgabedateinamen). {0} Parameter wurden übergeben: "{1}".</target>
         <note>{StrBegin="MSB4234: "}This is shown when the Binary Logger is passed zero or more than one parameter.</note>
       </trans-unit>
       <trans-unit id="IncludeRemoveOrUpdate">
         <source>MSB4232: Items that are outside Target elements must have one of the following operations: Include, Update, or Remove.</source>
-        <target state="new">MSB4232: Items that are outside Target elements must have one of the following operations: Include, Update, or Remove.</target>
+        <target state="translated">MSB4232: Elemente außerhalb der Zielelemente müssen einen der folgenden Vorgänge aufweisen: Einschließen, Aktualisieren oder Entfernen.</target>
```

Comment:
`Zielelemente` and `Einschließen, Aktualisieren oder Entfernen` shouldn't have been translated, see note below.

Looks like this mistake was made for a few of the other languages too.